relay-event-normalization/src/event.rs

```diff
@@ -270,11 +274,13 @@ fn normalize(event: &mut Event, meta: &mut Meta, config: &NormalizationConfig) -
     normalize_breakdowns(event, config.breakdowns_config); // Breakdowns are part of the metric extraction too
 
     processor::apply(&mut event.request, |request, _| {
-        request::normalize_request(request)
-    })?;
+        request::normalize_request(request);
+        Ok(())
+    })
+    .ok();
```

Comment:
There are a few places where this pattern is repeated (vs just a function call like above), but this is out of scope of this PR.